save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TagService.kt

```diff
@@ -33,6 +36,8 @@ class TagService(
      */
     @Transactional
     fun addVulnerabilityTag(identifier: String, tagName: String): LnkVulnerabilityMetadataTag {
+        log.info("Trying to add $tagName to $identifier vulnerability")
```

Comment:
```suggestion
        log.info { "Trying to add $tagName to $identifier vulnerability" }
```

Reply:
whyyyyyyyy?

Reply:
it's not idiomatic Kotlin, only Unit-like parameters should have it curly brackets

Reply:
because now string is built each time, with usign lambda -- it will be calculated only when info is enabled

Reply:
ah, we have utils for it